tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -167,7 +167,15 @@ Given('the following files have been deleted by user {string}', async function (
 })
 
 When('the user uploads file {string} using the webUI', function (element) {
-  return client.page.filesPage().uploadFile(element)
+  const uploadPath = path.join(client.globals.mountedUploadDir, element)
```

Comment:
Can't we hide this inside the uploadFile() function?

Reply:
It's not the concern of the `page`. From my POV, `page` should only care about setting *path* of the file/folder, nothing else. What if we need to be able to upload the file that is in `/tmp` directory or `session` uploads directory?